backend/api/Areas/Property/Models/Search/PropertyFilterModel.cs

```diff
@@ -73,6 +73,12 @@ public class PropertyFilterModel : PageFilter
         /// <value></value>
         public bool? IgnorePropertiesInProjects { get; set; }
 
+        /// <summary>
+        /// get/set - Flag indicating bare land is the desired return.
+        /// </summary>
+        /// <value></value>
+        public bool? BareLandOnly { get; set; }
```

Comment:
See above comment

Reply:
Remove and use the `PropertyType` instead.